packages/shopify-codemod/test/fixtures/coffeescript-soak-to-condition/assignment.output.js

```diff
@@ -0,0 +1,16 @@
+if (typeof bar === 'function' && bar() != null && bar().baz.qux != null) {
```

Comment:
Yeah, gave it some thought and I think that bailing out when we encounter this scenario is the best choice.